h2o-dist/index.html

```diff
@@ -258,23 +258,23 @@
       
       .callout--warning {
```

Comment:
nit pick: With the blue color, I recommend changing the class name from `.callout--warning` to `.callout--info`

Reply:
ah, good point. Done 👍